773290 - crashes on using combo box in import dialog

commit b313621b783a20281c32ccc11625bf08d6f88048
Author: etxrab <etxrab@f5534014-38df-0310-8fa8-9805f1628bb7>
Date:   Fri Jan 13 06:22:56 2012 +0000

    From  Edward Sheldrake:
    Wireshark crashes if you open the file import dialog and choose something from the "Encapsulation type" list.
    
    https://bugs.wireshark.org/bugzilla/show_bug.cgi?id=6729
    
    git-svn-id: http://anonsvn.wireshark.org/wireshark/trunk@40448 f5534014-38df-0310-8fa8-9805f1628bb7

diff --git a/gtk/file_import_dlg.c b/gtk/file_import_dlg.c
index 5ee9f3f..1d7535d 100644
--- a/gtk/file_import_dlg.c
+++ b/gtk/file_import_dlg.c
@@ -203,7 +203,7 @@ encap_co_changed(GtkComboBox *widget, gpointer data)
 
     if (result)
     {
-        guint8 encap;
+        guint encap;
         GtkTreeModel *model = gtk_combo_box_get_model(widget);
         gtk_tree_model_get(model, &iter, 1, &encap, -1);
 
